x-pack/plugins/observability/README.md

```diff
@@ -0,0 +1,19 @@
+# Observability Shared Resources
+
+This "faux" plugin serves as a place to statically share resources, helpers, and components across observability plugins. There is some discussion still happening about the best way to do this, but this is one suggested method that will work for now and has the benefit of adopting our pre-defined build and compile tooling out of the box.
+
+Files found here can be imported from any other x-pack plugin via:
+
+// for a file found at `x-pack/plugins/infra/public/components/Example.tsx`
+
+```ts
+import { ExampleSharedComponent } from '../../../observability/public/components/example_shared_component';
```

Comment:
Did the platform team agree to lift the restiction (by convention) to only statically import from a plugin's server-side or client-side entry `index.ts`?

Reply:
Good point, now that you say it I think they may have mentioned something about this. I’ll confirm.